Orange-OpenSource/Orange-Boosted-Bootstrap
# Release 5.3.0 🚀

- Firefox ESR
 - IE11 (v4 only)
 - Latest Edge, Chrome, Firefox, Safari
 - iOS Safari
 - Chrome & Firefox on Android
 - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) and `conventional-changelog-cli` globally
 - run `conventional-changelog -p angular -i CHANGELOG.md -s`
 - and probably maintain [a ship list (eg for v4.4.0)](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/226)
 - ~~Manually run BrowserStack test~~
 - attach zip file
 - paste CHANGELOG / Ship list in the release's description
 - `npm pack`
 - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
 - Publish:
 - if you're releasing a pre-release, use `--tag`, eg for v5-alpha1 `npm publish boosted-5.0.0-alpha1.tgz --tag next`
 - (v4 only) `npm publish --tag v4.x.y` (if you forgot and v4 becomes the latest version on NPM, you can run `npm dist-tag add boosted@5.x.y latest to fix it)
 - (v5 only) `npm publish`
